.github/workflows/no-banned-git-tags.yml

```diff
@@ -18,8 +18,8 @@ jobs:
   delete:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: fregante/setup-git-token@v1
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-    - run: git push --delete origin $GITHUB_REF
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-token@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you prefer these three lines or:

```yml
    - run: git config user.name "GitHub Actions" && git config user.email "actions@users.noreply.github.com"
```

Soft deprecation: https://github.com/fregante/setup-git-token

Reply:
I prefer your action until https://github.com/actions/checkout/issues/13 is resolved.